tests/bluechi_test/container.py

```diff
@@ -157,3 +157,15 @@ def enable_unit(self, node_name: str, unit_name: str) -> None:
         result, output = self.exec_run(f"bluechictl enable {node_name} {unit_name}")
         if result != 0:
             raise Exception(f"Failed to enable service {unit_name} on node {node_name}: {output}")
+
+    def run_python(self, python_script_dir: str, python_script_filename: str) -> \
+            Tuple[int | None, Iterator[bytes] | Any | Tuple[bytes, bytes]]:
+
+        source_file_path = os.path.join(python_script_dir, python_script_filename)
+        content = read_file(source_file_path)
+
+        target_file_dir = os.path.join("/", "usr", "bin")
```

Comment:
Not sure `/usr/bin` is the right place to store python scripts. Since you create it on every call, shouldn't it be stored in a temporary place and maybe even removed once the script is done?

Reply:
Good point! Changed it to just `/tmp` with a try-wrapped `os.remove`.